<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAWaitUntilActionWidget</class>
 <widget class="QWidget" name="GTAWaitUntilActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Parameter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ParamNameLE"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="SearchPB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>118</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Is</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="OperatorCB">
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NOT=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Value/Parameter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ValueLE">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="SearchValuePB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="checkFsWarLB">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Note: Check is only applicable for Bench parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="checkValnFS">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Note: Both Value and FS are checked if nothing is selected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QCheckBox" name="ChkLoopSampling">
       <property name="text">
        <string>Loop sampling</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="ChkValueOnly">
       <property name="text">
        <string>Check Only Value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="ChkFSOnly">
       <property name="text">
        <string>Check Only FS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkRefreshRate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Check Refresh Rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="chkOnlyParity">
       <property name="text">
        <string>Check only Parity</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="CheckSDICB">
       <item>
        <property name="text">
         <string>00</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>01</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="FunctionalStatusCB"/>
     </item>
     <item row="1" column="3">
      <widget class="QCheckBox" name="chkOnlySDI">
       <property name="text">
        <string>Check only SDI</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="CheckParityCB">
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="LoopSamplingLE"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="LoopSamplingCB">
       <item>
        <property name="text">
         <string>sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>msec</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ParamNameLE</tabstop>
  <tabstop>SearchPB</tabstop>
  <tabstop>OperatorCB</tabstop>
  <tabstop>ValueLE</tabstop>
  <tabstop>SearchValuePB</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
